app/services/concerns/xccdf/rule_group_relationships.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module Xccdf
+  # Methods related to saving RuleGroupRelationships
+  module RuleGroupRelationships
+    extend ActiveSupport::Concern
+    included do
+      def save_rule_group_relationships
+        @op_rules_and_rule_groups = @op_rule_groups + @op_rules
+        @rule_group_relationships = all_rule_group_relationships
+
+        ::RuleGroupRelationship.import!(
+          @rule_group_relationships.select(&:new_record?), ignore: true
+        )
```

Comment:
Why do you have conflict resolving (on new records) for `RuleGroupRules` but not here?

Reply:
I didn't do the conflict resolving here because there can be multiple required/conflicting rules and rule groups for a rule or rule group. So if there's already a `rule` that's requiring a certain `rule_group`, and then a new record comes in where that `rule` is requiring a different `rule_group`, we wouldn't want to update the current record in any way because it might still be valid, since it can require multiple rules/rule groups
This is why I thought it was best to just ignore duplicates. Let me know if this makes sense!

Reply:
Makes sense, just inconsistent with [this](https://github.com/RedHatInsights/compliance-backend/pull/1143#discussion_r821515595)...